app/utils/answersHelpers/SummaryListRowHelper.scala

```diff
@@ -55,6 +64,9 @@ class SummaryListRowHelper(implicit messages: Messages) {
   def formatAsDate(answer: LocalDate): Content =
     answer.format(cyaDateFormatter).toText
 
+  def formatAsDate2(answer: XMLGregorianCalendar): Content =
```

Comment:
Can this not also be called `formatAsDate`? Should just overload it

Reply:
done